addons/sup_combatsupport/fnc_getUserMenuButton.sqf

```diff
@@ -18,6 +18,7 @@ diag_log format["MenuButton: %1", _return];
     {
     _return = getNumber(configfile >> "ALiVE_UserConfig" >> _var)
     };
-    call compile format ["%1 = %2", _var,_return];
+    missionNamespace setVariable [_var,_return,true];
```

Comment:
where does the public variable come from all of the sudden?

Reply:
Resolved in next commit